components/movementsensor/collectors.go

```diff
@@ -4,12 +4,42 @@ import (
 	"context"
 	"errors"
 
+	v1 "go.viam.com/api/common/v1"
+	pb "go.viam.com/api/component/movementsensor/v1"
 	"google.golang.org/protobuf/types/known/anypb"
 
 	"go.viam.com/rdk/data"
 )
 
 // TODO: add tests for this file.
+type method int64
+
+const (
+	position method = iota
+	linearVelocity
+	angularVelocity
+	compassHeading
+	linearAcceleration
+	orientation
+)
+
+func (m method) String() string {
+	switch m {
+	case position:
+		return "Position"
```

Comment:
What are these strings used for? Should they match the method names in the API?

Reply:
I'd be in favor of them matching the API method names, but agreed that it's a separate change

Reply:
Filed [here](https://viam.atlassian.net/browse/DATA-2018)!